ui/src/elements/ngm-cam-configuration.ts

```diff
@@ -292,21 +325,30 @@ export class NgmCamConfiguration extends LitElementI18n {
             <div class=${c.iconClass()} title=${i18next.t('cam_lock')} @click=${c.lock}></div>
             <div class="ngm-cam-conf-slider">
               <div>
-                <label>${unsafeHTML(c.label())}</label>
-                <label>${c.getValueLabel()}</label>
+                <label>${c.label()}</label>
+                <input type="number" class="ngm-cam-conf-number-input"
+                      min="${c.minInputValue ?? c.minValue}"
+                      max="${c.maxInputValue ?? c.maxValue}"
+                      step="${c.inputStep ?? c.step}"
+                     .value=${c.getInputValue ? c.getInputValue() : c.getValue()}
+                     @input=${c.onInputChange ? c.onInputChange : c.onSliderChange}
+
+               />
               </div>
               <input type="range" class="ngm-slider" style=${styleMap(c.style())}
                      min=${c.minValue}
                      max=${c.maxValue}
                      step=${c.step}
                      .value=${c.getValue()}
-                     @input=${c.onChange}/>
+                     @input=${c.onSliderChange}
+                     @keydown="${(e) => { if (e.key === 'ArrowLeft' || e.key === 'ArrowRight') e.stopPropagation(); }}"
```

Comment:
Why is this needed?

Reply:
Because the map also moved when changing the values of the sliders

Reply:
I thought as much, but I tried it without this event listener & everything seemed fine to me. Might be a browser-specific issue then.